### DISCUSSION TITLE
How can I use dynamic import for a package that uses import aliases?

### DISCUSSION THREAD

**Discussion by raulbarriga** [vercel/next.js#28191] [Help]
I have to import a package like this `import { Fade as Hamburger } from "hamburger-react`. I also have to use dynamic import since the packge won't work for serv side rendering. 

I've seen examples of using dynamic imports for a package that's only for the browser. But none address if the package requires using import aliases for the components.

**Comment by wwwdepot** [comment]
Use With named exports https://nextjs.org/docs/advanced-features/dynamic-import#with-named-exports
```js
import dynamic from 'next/dynamic'
const Hamburger = dynamic(() => import('hamburger-react').then(mod => mod.Fade));
```
